Source/headless/HeadlessFontInfo.m

```diff
@@ -0,0 +1,86 @@
+/*
+   HeadlessFontInfo.m
+   Copyright (C) 2003, 2023 Free Software Foundation, Inc.
+   August 17, 2023
+   Re-wrtten by Gregory Casamento <greg.casamento@gmail.com>
+   Based on work by Marcian Lytwyn <gnustep@advcsi.com>
+   August 31, 2003
+   Written by Banlu Kemiyatorn <object at gmail dot com>
+   Base on original code of Alex Malmberg
+   This file is part of GNUstep.
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
+   Lesser General Public License for more details.
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; see the file COPYING.LIB.
+   If not, see <http://www.gnu.org/licenses/> or write to the
+   Free Software Foundation, 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110-1301, USA.
+*/
+
+#include "GNUstepBase/Unicode.h"
+#include <AppKit/NSAffineTransform.h>
+#include <AppKit/NSBezierPath.h>
+#include "headless/HeadlessFontInfo.h"
+#include "headless/HeadlessFontEnumerator.h"
+
+#include <math.h>
+
+@implementation HeadlessFontInfo
+
+- (id) initWithFontName: (NSString *)name
+		 matrix: (const CGFloat *)fmatrix
+	     screenFont: (BOOL)p_screenFont
+{
+  self = [super init];
+  if (!self)
+    return nil;
+
+#ifndef _MSC_VER
+  // Accessing instance variables across module boundaries is not supported by the Visual Studio
```

Comment:
If this is a real issue, we need a better solution for it. Maybe implement a simple version of this method on GSFontInfo in gui and call that form back. But this could be resolved independently of you pull request.

Reply:
Yes, the proper fix for this is likely getter/setter methods for a bunch of instance variables in libs-gui.  Getting the Win32 backend to compile on msvc will require similar kind of work.  I hope to get to that one day, but it will be on a separate PR ;-)